src/db/migrations/index.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const path = require('path');
+
+let executedMigrations = [];
+
+if (fs.existsSync('executedMigration.json')) {
```

Comment:
All migration manipulation should be done in `migrate` function (including variable `executedMigrations`).